test/correctness/fast_arctan.cpp

```diff
@@ -0,0 +1,87 @@
+#include "Halide.h"
+
+#ifndef M_PI
+#define M_PI 3.14159265358979310000
```

Comment:
Nit: just declare what you want here, eg `const double Pi = 3.14159265358979310000;`

Reply:
Can do that, but I was just copying from `fast_cosine_sine.cpp`.

Reply:
Oh, it's also unused...